### DISCUSSION TITLE
Not all delivery customizations run when discount code is removed

### DISCUSSION THREAD

**Discussion by Sam115200** [Shopify/function-examples#369] [Feedback: Delivery Customization API]
**Setup:**

2 delivery customations are enabled on the store, the first one arranges the shipping rates from high to low price. The second one hides "Standard" shipping rate when the cart total is below $1500. 

Initially both customizations work fine, as the cart total is above $1500. Then a discount is applied causing Standard shipping to be hidden. When discount code is removed, then Standard shipping is shown again but at the bottom of the list, which is wrong.

I thought the sort customization might have ran before the hide customization when discount code is removed, but creating the hide customization first and then sort customizarion second, also did not help. No matter, the Standard shipping is always shown at the bottom of the list.

**Video:**
https://drive.google.com/file/d/1Ah4LHBXOudQyCFAkSz6rEKEvd5P81-2D/view?usp=sharing

**Checkout Link:**
https://storespark-co-ext.myshopify.com/checkouts/cn/c1-d6db0b8f311e55644fb83142d304807b

**Expected:**
Sort customization should also run when discount code is removed at the checkout.

**Comment by sebastianpisula** [comment]
+1
